lib/trace.js

```diff
@@ -249,12 +249,12 @@ const traceFile = async ({
         .catch((err) => {
           // Handle module not found.
           if (err.code === "MODULE_NOT_FOUND") {
-            // If allowed, then totally ignore.
+            // If allowed, then add to depObjs anyway.
             const srcPkg = getLastPackage(srcPath);
             const isAllowed = srcPkg && (allowMissing[srcPkg] || [])
               .some(matchesPkgPrefix(depName));
             if (isAllowed) {
-              return null;
```

Comment:
By returning null here the package will be filtered in line 284, which will cause line 286 to return true, causing standard Node.js libraries to not be filtered.